unittests_gl/uniformbuffer.cpp

```diff
@@ -0,0 +1,213 @@
+/*****************************************************************************
+ * Alpine Renderer
+ * Copyright (C) 2022 Adam Celarek
```

Comment:
copyright!

if you didn't replace all of the file, keep jakobs and my copyright. there is no point in checking git log, who owns which lines. having several copyrights more than necessary hurts less than forgetting about one. it would be super bad if at some point sb comes and says, hey, i wrote that code. where is my copyright :)